cli/tests/unit/stat_test.ts

```diff
@@ -9,7 +9,7 @@ import {
 
 Deno.test({ permissions: { read: true } }, function fstatSyncSuccess() {
   const file = Deno.openSync("README.md");
-  const fileInfo = Deno.fstatSync(file.rid);
```

Comment:
I'm wondering if we should keep a some tests to make sure deprecated APIs work until Deno 2? This probably applies to all deprecation PRs and the ones merged already

Reply:
I agree, and I've done (or at least aimed to be conscious of it) for the other PRs. This one just slipped.